src/features/deleteCompany/deleteCompanyApiSlice.ts

```diff
@@ -46,7 +46,7 @@ export const apiSlice = createApi({
     }),
     declineRegistration: builder.mutation<void, string>({
       query: (applicationId) => ({
-        url: `/api/registration/application/${applicationId}/declineregistration`,
+        url: `/api/registration/applications/${applicationId}/declineRegistration`,
```

Comment:
why are you changing again to camelcase for declineRegistration, you just changed it: https://github.com/eclipse-tractusx/portal-frontend/pull/831

Reply:
Fix mentioned in the task for "s" and not for camel case. So reverted the change. @jjeroch / @Phil91  can anyone confirm this?

Reply:
@jjeroch also mentioned the endpoint in the issue ` /api/registration/applications/{applicationId}/declineregistration`, camelcase is wrong as well, IMO

Reply:
But when I spoke to her, she mentioned that fix is to add "s" to the application and not camel case change. Will wait for the confirmation. I can change it from camel to all small case

Reply:
@evegufy @manojava-gk the documentation is not 100% correct there as the example in line 161 doesn't match the  path in line 169

https://github.com/eclipse-tractusx/portal-backend/blob/a0b4b0d5729252bdf5b012d4cd592789092b68aa/src/administration/Administration.Service/Controllers/RegistrationController.cs#L161-L169

Reply:
@manojava-gk @oyo the correct url is /declineregistration

see [RegistrationController](https://github.com/eclipse-tractusx/portal-backend/blob/42e73066a27fb5a7343d4b08459f832eb9334c85/src/registration/Registration.Service/Controllers/RegistrationController.cs#L510) as reference

Reply:
@Phil91 yes all lowercase, but still there is still an example in the documentation line 500 that doesn't match the path in line 507 (missing 's' after application)

[RegistrationController.cs#L507](https://github.com/eclipse-tractusx/portal-backend/blob/release/v2.0.0-RC10/src/registration/Registration.Service/Controllers/RegistrationController.cs#L507)

`[Route("applications/{applicationId}/declineregistration")]`

[RegistrationController.cs#L500](https://github.com/eclipse-tractusx/portal-backend/blob/release/v2.0.0-RC10/src/registration/Registration.Service/Controllers/RegistrationController.cs#L500)

```diff
- /// Example: POST: /api/registration/application/{applicationId}/declineregistration
+ /// Example: POST: /api/registration/applications/{applicationId}/declineregistration
```